newrelic/newrelic.go

```diff
@@ -115,9 +119,12 @@ func ConfigAdminAPIKey(adminAPIKey string) ConfigOption {
 }
 
 // ConfigRegion sets the New Relic Region this client will use.
-func ConfigRegion(r region.Name) ConfigOption {
+func ConfigRegion(r string) ConfigOption {
```

Comment:
Could rename the method for clarity.  `ConfigureRegion()` perhaps?

Reply:
I think we'd want to rename all of our config methods for consistency if we wanted to do this here.  How strongly do you feel about this?

Reply:
Let's talk about adding this as a separate PR.